angrmanagement/config/config_manager.py

```diff
@@ -57,6 +57,9 @@ def color_constructor(config_option, value):
     CE('feature_map_color_unknown', QColor, QColor(0xa, 0xa, 0xa)),
     CE('feature_map_color_delimiter', QColor, QColor(0, 0, 0)),
     CE('feature_map_color_data', QColor, QColor(0xc0, 0xc0, 0xc0)),
+    # plugins
+    CE('plugin_search_path', str, '$BUILTIN_PLUGINS:~/.local/share/angr-management/plugins'),
```

Comment:
Could we make this `AM_BUILTIN_PLUGINS` or something slightly less generic? This literally collides with another project of mine.

Reply:
fixed. that's embarassing...